core/templates/dev/head/player/PlayerServices.js

```diff
@@ -634,6 +634,21 @@ oppia.factory('ratingService', [
   };
 }]);
 
+oppia.factory('speechService', ['oppiaHtmlEscaper', function(oppiaHtmlEscaper) {
+  return {
+    getSpokenText: function(component) {
+      var componentId = component.nodeName;
+      if (componentId == "#text") {
+        return component.nodeValue;
+      }
+      if (componentId == "OPPIA-NONINTERACTIVE-LINK") {
+        var linkPreface = "Clickable Link. "
+        return linkPreface + oppiaHtmlEscaper.escapedStrToUnescapedStr(component.attributes.item(1).nodeValue);
```

Comment:
Any way to make this shorter aside from brackets and a line break after "return"?